for_challenges.py

```diff
@@ -25,6 +29,11 @@
     'Маша': False,
 }
 names = ['Оля', 'Петя', 'Вася', 'Маша']
+for name in names:
+    if is_male[name]:
```

Comment:
Тут можно чуть сократить код. Как минимум нам хотелось бы что бы осталась только одна f-строка, что бы избежать двойного редактирования. 
```suggestion
    gender = 'мужской' if is_male[name] else 'женский'
    print(f'{name} {gender}')
```